gprofiler/merge.py

```diff
@@ -92,33 +176,39 @@ def parse_perf_script(script: str):
             match = SAMPLE_REGEX.match(sample)
             if match is None:
                 raise Exception("Failed to match sample")
-            yield match.groupdict()
+            sample_dict = match.groupdict()
+
+            pid = int(sample_dict["pid"])
+            process_name = sample_dict["comm"]
```

Comment:
```suggestion
            comm = sample_dict["comm"]
```
Keep the logical naming please